src/react-components/room/ExitedRoomScreen.js

```diff
@@ -1,75 +1,110 @@
 import React from "react";
 import PropTypes from "prop-types";
-import IfFeature from "../if-feature";
-import { getMessages } from "../../utils/i18n";
-import configs from "../../utils/configs";
-import { FormattedMessage } from "react-intl";
+import { defineMessages, useIntl } from "react-intl";
+import { LoadingScreenLayout } from "../layout/LoadingScreenLayout";
+
+const messages = defineMessages({
+  exited: {
+    id: "exit.subtitle.exited",
+    defaultMessage: "Your session has ended. Refresh your browser to start a new one."
+  },
+  closed: {
+    id: "exit.subtitle.closed",
+    defaultMessage: "This room is no longer available."
+  },
+  denied: {
+    id: "exit.subtitle.denied",
+    defaultMessage: "You are not permitted to join this room. Please request permission from the room creator."
+  },
+  disconnected: {
```

Comment:
Would be good to distinguish "you disconnected" from "you've been kicked", even if the ability to rejoin the room is the same in both cases. (This isn't a new change with this PR)

Reply:
Agreed. There's also a comment in hub.js that mentions this: https://github.com/mozilla/hubs/blob/master/src/hub.js#L1134